common/src/main/scala/com/gu/recipeasy/models/Statistics.scala

```diff
@@ -20,3 +20,43 @@ case object PStatsBiggestDayCount extends PersonalStatisticsPoint
 case object PStatsRanking extends PersonalStatisticsPoint
 
 case class DayActivityDistribution(date: String, curationCount: Int, verificationCount: Int, confirmationCount: Int)
+
+case class LeaderboardEntry(userEmail: String, curationCount: Int, verificationCount: Int, confirmationCount: Int) {
+  def total(): Int = {
+    curationCount + verificationCount + confirmationCount
+  }
+}
+
+class Leaderboard(data: List[LeaderboardEntry]) {
```

Comment:
Yeah. Make sense. In fact the type of the argument was a late change and I didn't try to case class it afterward (was not possible before). Give me two seconds....